gossip/src/cluster_info.rs

```diff
@@ -983,7 +983,7 @@ impl ClusterInfo {
         assert!((vote_index as usize) < MAX_LOCKOUT_HISTORY);
         let self_pubkey = self.id();
         let now = timestamp();
-        let vote = Vote::new(self_pubkey, vote, now);
+        let vote = Vote::new(self_pubkey, vote, now).unwrap();
```

Comment:
Any concerns with externally controlled info making it to this unwrap?

Reply:
I don't think so. Unless replay is pushing invalid votes.
These are local nodes votes.